frontend/src/pages/WorkspaceTeam/components/AutoJoinForm.tsx

```diff
@@ -103,7 +107,8 @@ function AutoJoinForm({
 			<div className={styles.container}>
 				<Switch
 					trackingId="WorkspaceAutoJoin"
-					label="Enable Auto Join"
+					label={label || 'Enable Auto Join'}
+					labelFirst={labelFirst}
```

Comment:
I'm not seeing where `labelFirst` is being used in this PR 🤔 - is it used, or something we wanted to have optionality to use in the future?

Reply:
Thanks for calling out! I just added it [above](https://github.com/highlight/highlight/pull/5425/files#diff-15b316d6b516d917b0b67d03ed4b796a8d019214479d09a12be65be4a607efb6R124) for the new instance.